### DISCUSSION TITLE
how to plot spectrum correctly

### DISCUSSION THREAD

**Discussion by lxdmay** [svenreiche/Genesis-1.3-Version4#192] [Q&A]
hello, Sven, I use Example3.py to calculate the spectrum. The center wavelength of the drawn result is 1.2nm, but the lambda0 (resonant wavelength) I set is 1nm, so I don't know where the problem is?
![image](https://github.com/user-attachments/assets/edaed245-aef0-4fa9-9796-dc78e9c74420)
![image](https://github.com/user-attachments/assets/4fb39d4a-d61e-4fb4-bfac-beeb92a607ac)

**Comment by svenreiche** [comment]
There was recently a bug fixed for the correct spectral information. But let me check it.

**Reply by lxdmay** [reply]
OK，thank you.
